src/benchmarks/micro/MicroBenchmarks.csproj

```
@@ -109,6 +109,10 @@
     <Compile Remove="libraries\System.Formats.Cbor\*.cs" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' Or ('$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(_TargetFrameworkVersionWithoutV)' &lt; '6.0')">
```

I wonder whether it would be possible to simplify the condition to just `<= 6.0` check:

```suggestion
  <ItemGroup Condition=" '$(_TargetFrameworkVersionWithoutV)' &lt; '6.0' ">
```

---

I just commited it, let's see if CI works.